libraries/browser-tracker-core/test/browser_props.test.ts

```diff
@@ -1,14 +1,17 @@
-import { floorDimensionFields, getBrowserProperties } from '../src/helpers/browser_props';
+import { makeDimension, getBrowserProperties } from '../src/helpers/browser_props';
 
 describe('Browser props', () => {
-  it('floorDimensionFields correctly floors dimension type values', () => {
+  it('makeDimension correctly floors dimension type values', () => {
```

Comment:
Since the case we fixed was about none I would just add one more test with NaN values